### DISCUSSION TITLE
xload扩展讨论

### DISCUSSION THREAD

**Discussion by wujinpai** [1Panel-dev/1Panel#4372] [Q&A]
### 1Panel 版本

1.10.2

### 请描述您的需求或者改进建议

![IMG_2526](https://github.com/1Panel-dev/1Panel/assets/43784195/9a0a82a1-f088-4939-b9c8-2f2aca8a8ac5)
![IMG_2527](https://github.com/1Panel-dev/1Panel/assets/43784195/56e7a8e8-a8f2-4072-ab8c-b11b453978d1)

我去问过xload插件制作官方了，也把1panel面板的账户密码发给他测试了一下，他邮件给了回复，如上图

**Comment by wujinpai** [comment]
![image](https://github.com/1Panel-dev/1Panel/assets/43784195/49ccdae9-b6d8-4f52-9e8b-25c3e5a69c8e)![image](https://github.com/1Panel-dev/1Panel/assets/43784195/9c758fd4-e640-4b20-9136-4b70459f53f4)

**Comment by wanghe-fit2cloud** [comment]
确实需要咨询 xload 官方容器化使用教程，你可以在已安装应用页面查看路径。
1Panel 负责容器化安装 PHP 运行环境，包括官方支持的扩展，并提供自定义扩展的教程（例如swoole_loader）。
如果 xload 仍然无法正确安装，可能需要先咨询其官方支持，因为 1Panel 这边也不是很了解其情况。

**Reply by wujinpai** [reply]
上面的邮件回复就是xload官方的

**Reply by wanghe-fit2cloud** [reply]
已安装应用列表查看应用安装路径
